adafruit_ble_berrymed_pulse_oximeter/adafruit_ble_transparent_uart.py

```diff
@@ -55,7 +59,7 @@ def __init__(self, service=None):
             self._tx = self._server_rx
             self._rx = self._server_tx
 
-    def read(self, nbytes=None):
+    def read(self, nbytes: Union[bytes, None] = None) -> Union[bytes, None]:
```

Comment:
Same thing here, `Optional` is the way to go, even as a return type:

```suggestion
def read(self, nbytes: Optional[bytes] = None) -> Optional[bytes]:
```